### ISSUE TITLE
Failed to load AppCenter deployment history (The app x/x does not exist. Please double check the name, and provide it in the form owner/appname.)

### ISSUE THREAD

**Issue by mrousavy** [getsentry/sentry-cli#904]
### Environment

We're using sentry SaaS with React-Native (latest).

I tried to follow the [using sentry with codepush](https://docs.sentry.io/platforms/react-native/codepush/) guide, but can't seem to upload the codemaps generated by codepush to Sentry:

![Screenshot 2021-02-10 at 13 43 40](https://user-images.githubusercontent.com/15199031/107511600-fcccfb00-6ba5-11eb-9826-b8e1abccae11.png)

As you can see, it can't find the app `Cuvent/Cuvent-iOS`, but when I list all my apps using `appcenter apps list` it is definitely there:

![Screenshot 2021-02-10 at 13 44 06](https://user-images.githubusercontent.com/15199031/107511640-0c4c4400-6ba6-11eb-9d95-86baa260a307.png)

### Steps to Reproduce

Basically just follow the [using sentry with codepush](https://docs.sentry.io/platforms/react-native/codepush/) guide.

1. Create codepush update and output sourcemaps in `./build`
2. Try to upload those maps using `sentry-cli react-native appcenter {APP} ios ./build/CodePush --deployment {DEPLOYMENT} --dist {DIST}`
3. Notice how that command fails because it couldn't find the App

### Expected Result

I expect it to upload the sourcemaps from the codepush update successfully

### Actual Result

It couldn't find the app ([404'd in this piece of code](https://github.com/microsoft/appcenter-cli/blob/master/src/commands/codepush/deployment/list.ts#L38-L58))

**Comment by pschoen325** [comment]
This resolved the issue for me. Note that the --deployment value has to match your codepush deployment exactly.
https://github.com/getsentry/sentry-cli/issues/415#issuecomment-673325836
